### PR TITLE
K5.0: Crypsis poll datepicker can't be opened(fix for boostrap 2) #3694

### PR DESCRIPTION
**Pull request by xillibit**
Pull Request for Issue #3694.

### CODE REVIEW

**Comment by rich20** [comment]
- [ ] Failed - can't be opened

The console shows:
../media/kunena/cache/crypsis/css/css/bootstrap.datepicker.css 404 (Category not found)
../media/kunena/js/bootstrap.datepicker.js 404 (Category not found)

**Comment by xillibit** [comment]
I have renammed the files in wrong place, i will fix that later

**Comment by xillibit** [comment]
Re-try now because with this PR: https://github.com/Kunena/Kunena-Forum/pull/3702#event-568665804 it should works

**Comment by rich20** [comment]
> Re-try now because with this PR: https://github.com/Kunena/Kunena-Forum/pull/3702#event-568665804 it should works
- [ ] Failed - can't be opened

../components/com_kunena/template/crypsis/bootstrap-datepicker.js 404 (Category not found)

File: src/components/com_kunena/template/crypsis/layouts/topic/edit/default.php [hunk 1]
```diff
@@ -57,8 +57,8 @@
 	$this->addScript('poll.js');
 }
 
-$this->addStyleSheet('css/bootstrap-datepicker.css');
-$this->addScript('bootstrap-datepicker.js ');
+$this->addStyleSheet('css/bootstrap.datepicker.css');
```
Comment (810) [inline src/components/com_kunena/template/crypsis/layouts/topic/edit/default.php]: change to:

```
$this->addStyleSheet('assets/css/bootstrap.datepicker.css');
$this->addScript('assets/js/bootstrap.datepicker.js'); 
```

**Comment by 810** [comment]
@rich20 try this changes

**Comment by rich20** [comment]
+1

**Comment by xillibit** [comment]
Changes done

**Pull request merged**